libbeat/api/npipe/pipe.go

```diff
@@ -0,0 +1,25 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package npipe
+
+import "strings"
+
+// IsNPipe returns true if the string has a npipe scheme.
+func IsNPipe(s string) bool {
+	return strings.HasPrefix(s, "npipe:///")
```

Comment:
what's your opinion on supporting also `HasPrefix("//./pipe/")`?

Reply:
I am ok with supporting both, to be honest, I was not sure what would be the best way to represent them in a configuration, but supporting npipe + the real representation \\.\pipe\hello is ok.